pandas_gbq/gbq.py

```diff
@@ -557,7 +557,25 @@ def load_data(self, dataframe, dataset_id, table_id, chunksize):
 
         self._print("\n")
 
-    def verify_schema(self, dataset_id, table_id, schema):
+    def schema(self, dataset_id, table_id):
```

Comment:
can you update the ``to_gbq`` doc-string as well (with expl of the new behavior of if_exists)